cachix/src/Cachix/Client/Commands.hs

```diff
@@ -214,7 +214,7 @@ pushStrategy store authToken opts name compressionMode storePath =
       onAttempt = \retrystatus size -> do
         path <- decodeUtf8With lenientDecode <$> storePathToPath store storePath
         -- we append newline instead of putStrLn due to https://github.com/haskell/text/issues/242
-        putStr $ retryText retrystatus <> "compressing and pushing " <> path <> " (" <> humanSize (fromIntegral size) <> ")\n",
+        putStr $ retryText retrystatus <> "compressing using " <> T.toLower (show compressionMode) <> " and pushing " <> path <> " (" <> humanSize (fromIntegral size) <> ")\n",
```

Comment:
Could we show the compression settings once at the very top?

Reply:
The level or did you have something else in mind?

Reply:
Instead of printing out the compression mode for every path, we could print out the compression mode (and any other useful info) at the top. Then this line becomes much less verbose: `Pushing <path> (<size>)`.